lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -150,15 +163,16 @@ Repair query key to set the values if we can repair
 
 =cut
 
-has id_where => (is => 'ro', isa => 'Str', required => 0, default => 'id');
+has id_where => (is => 'ro', isa => 'Str', required => 0, default => '');
```

Comment:
Now that we support complex primary keys, what value does this attribute still have?

Reply:
None. Either id_where or id_columns can be used. Both cannot be empty though.

Reply:
I don't understand why we need `id_where` in case we already have `id_columns`